frontend/src/components/auth/LoginForm.vue

```diff
@@ -1,62 +1,72 @@
 <template>
-    <VContent>
-        <VFlex
-            lg6
-            md6
-            sm12
-            xs12
-            :class="{'mx-5': $vuetify.breakpoint.smAndUp}"
+    <div class="form">
+        <Spinner
```

Comment:
On the meeting, we came up with a decision not to block a whole form, but only a button